app/models/user.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+module.exports = (sequelize, DataTypes) => {
+  const User = sequelize.define(
+    'User',
+    {
+      id: {
+        type: DataTypes.INTEGER,
+        primaryKey: true,
+        unique: true,
```

Comment:
As this `id` is the primary key of the table, it is `unique` by default, so this is not needed